Project/Project/Controllers/AuthController.cs

```diff
@@ -63,9 +63,10 @@ public async Task<IActionResult> Login(LoginViewModel model, string returnUrl =
             var newestRecommendationDate = await _recommendationService.GetNewestRecommendationDate(userId);
             var favorites = await _recommendationService.GetFavorites(userId);
             var ratings = await _recommendationService.GetLikedMovies(userId);
+            var recommendationCount = await _context.Recommendations.CountAsync(r => r.User_id == userId);
             
             if ((favorites.Any() || ratings.Any()) &&
-                (newestRecommendationDate == null || newestRecommendationDate < DateTime.Now.AddDays(-7)))
+                ((newestRecommendationDate == null || newestRecommendationDate < DateTime.Now.AddDays(-7)) || recommendationCount < 5 ))
```

Comment:
I think this is not working properly. First i got these recommendations, which is ok:
<img width="1280" alt="image" src="https://github.com/user-attachments/assets/7b7bedae-6bf6-4cc6-8b30-55bda957bc18">

and then on my next login, as i had less than 5 recommendations i got new ones, but i got duplicate recommendations, like The Dark Knight Rises and Black Panther
<img width="671" alt="image" src="https://github.com/user-attachments/assets/48e4f258-2bca-40b6-9182-b85f1a6976f2">

Reply:
Couldnt replicate you error but made sure you dont get duplicate now, the amount on second try is fine (if less than 4 then request new ones, no 13 at max now)